picasso/templates/PartialOcclusion.html

```diff
@@ -26,9 +26,9 @@
 						<img src="inputs/{{ result.filename }}" style="width:244px;height:244px;"/>
 					</td>
 					<td align="center">
-						<img src="outputs/{{ result.example_filename }}" style="width:244px;height:244px;"/>
+						<img src="outputs/{{ result.processed_input_file_name }}" style="width:244px;height:244px;"/>
 					</td>
-					{% for filename in result.result_filenames %}
+					{% for filename in result.output_file_names %}
```

Comment:
Will need to update the documentation for "Making your own visualizations" to reflect this change.  Should just be a search and replace.

Reply:
Oh okay, I assumed that PartialOcclusion is the odd one out since the other two visualisers are using the 'new' naming scheme.

Reply:
You may be right.  In that case can you just look at that documentation page and make sure the examples are consistent with the new naming scheme?

https://picasso.readthedocs.io/en/latest/visualizations.html

Reply:
Done! The documentation is in line with the spelling that I corrected here.